classes/core/PKPQueueProvider.inc.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * @file classes/core/PKPQueueProvider.inc.php
+ *
+ * Copyright (c) 2014-2021 Simon Fraser University
+ * Copyright (c) 2000-2021 John Willinsky
+ * Distributed under the GNU GPL v3. For full terms see the file docs/COPYING.
+ *
+ * @class PKPQueueProvider
+ * @ingroup core
+ *
+ * @brief Registers Events Service Provider and boots data on events and their listeners
+ */
+
+namespace PKP\core;
+
+use Illuminate\Queue\WorkerOptions;
+
+use PKP\config\Config;
+use PKP\Domains\Jobs\Job as PKPJobModel;
+
+class PKPQueueProvider
+{
+    public function runJobsAtShutdown(): void
+    {
+        $disableRun = Config::getVar('queues', 'disable_jobs_run_at_shutdown', false);
+
+        if ($disableRun) {
+            return;
+        }
+
+        $job = PKPJobModel
+            ::isavailable()
```

Comment:
Is this `alllowercase` a Laravel convention? Generally we use `lowerCamelCase` for function naming.